src/components/notifcations/Notifications.tsx

```diff
@@ -0,0 +1,43 @@
+import { UserModel } from "../../lib/schemas/user.schema";
+
+export async function addNotification(userId: string, newNotification: string) {
+    const user = await UserModel.findById(userId);
+    if (user) {
+      if (user.notifications.length >= 3) {
+        user.notifications.shift(); // Remove the oldest notification
+      }
+      user.notifications.push(newNotification); // Add the new notification
+      await user.save();
+    }
+  }
+
+export default async function displayNotifications(userId: string | undefined){
```

Comment:
Is this a React component? If it is we need to rename this to conform to React component naming conventions.